Readme.md

```diff
@@ -16,6 +16,7 @@ On AArch64 hosts the user **MUST** have an x86-64 RootFS [Creating a RootFS](#Ro
 * clang (version 10 minimum for C++20)
 * libnuma-dev
 * libglfw3-dev (For GUI)
```

Comment:
Remove this, or is glfw still needed?

Reply:
glfw is still technically needed for a different GUI.